MRISimulation/src/CoilSensitivities.jl

```diff
@@ -28,3 +29,110 @@ function birdcageSensitivity(N::Int64,
   end
  return out
 end
+
+"""
+    measured2DSensitivity(N::Int64, ncoils::Int64)
+
+Computes 2D coil sensitivity maps in image space
+from actually measured coil profiles.
+The code was originally implemented in the
+Berkeley Advanced Reconstruction Toolbox (BART) by
+
+  M. Uecker et al. Software toolbox and programming library for compressed sens-
+  ing and parallel imaging. In: ISMRM Workshop on Data Sampling and
+  Image Reconstruction, Sedona, 2013.
+
+Checkout the original file:
+
+  https://github.com/mrirecon/bart/blob/master/src/simu/sens.c
+
+for more information.
+"""
+
+function get_sens_coeff(kx::Int64,
+                        ky::Int64,
+                        c::Int64)
```

Comment:
```suggestion
function get_sens_coeff(kx::Int,
                        ky::Int,
                        c::Int)
```
No reason to insist on 64 bit int is there?